setup.py

```diff
@@ -17,8 +17,12 @@ def read(fname):
     keywords = "modelica dymola openmodelica mat",
     url = "http://simulationresearch.lbl.gov/modelica/",
     install_requires = ['future',
+                        'gitpython',
+                        'jinja2',
+                        'matplotlib',
+                        'numpy',
                         'pytidylib',
-                        'gitpython'],
+                        'scipy'],
```

Comment:
What of your code requires `scipy`? This is a big dependencies to check the value of a tolerance.